app.rb

```diff
@@ -0,0 +1,18 @@
+require 'sinatra/base'
+require_relative 'lib/messages'
+
+  class Chitter < Sinatra::Base
+
+    get '/' do 
+      @messages = Messages.get_all
+      erb(:index)
+    end 
+
+    post '/' do 
+      Messages.create(message: params[:message],name: params[:name])
+      redirect '/'
+    end 
```

Comment:
Is this route RESTful? Perhaps this should be `post '/messages'` :thinking: 
In the index.erb file there is a form that posts to /messages even, is this a WIP? :thinking: